client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -575,6 +574,23 @@ export const ApplicationsTable: React.FC = () => {
         >
           {t("actions.delete")}
         </DropdownItem>,
+        ...(tasksReadAccess && tasksWriteAccess
+          ? [
+              <DropdownItem
+                key="applications-bulk-cancel"
+                isDisabled={
+                  !selectedRows.some((application: DecoratedApplication) =>
```

Comment:
We allow here a mix of cancellable and non-cancellable tasks. I think this is the right choice because the status may change quickly before user will be able to invoke the action.
Note however that this is different then our usual way of  handling bulk actions where ALL items need to match the criteria.

Reply:
@sjd78 ^^